tests/tck-build-logic/src/main/groovy/org.graalvm.internal.tck-harness.gradle

```diff
@@ -102,6 +103,51 @@ def matrixDefault = [
 
 final String METADATA_GROUP = "Metadata"
 
+List<String> getNewerVersionsFor(String library, String startingVersion) {
+    def baseUrl = "https://repo1.maven.org/maven2"
+    String group = library.split(":")[0].replace(".", "/")
+    String artifact = library.split(":")[1]
+    def data = new URL(baseUrl + "/" + group + "/" + artifact + "/" + "maven-metadata.xml").getText()
+
+    return tck.getNewerVersionsFromLibraryIndex(data, startingVersion, library)
+}
+
+// gradle getExistingLibrariesWithNewVersions
+Provider<Task> getExistingLibrariesWithNewVersions = tasks.register("getExistingLibrariesWithNewVersions", DefaultTask) { task ->
+    task.setDescription("Returns list of all libraries coordinates")
```

Comment:
with the latest versions? Or all coordinates possible?

Reply:
I thought the name of the task `getExistingLibrariesWithNewVersions` is explainable enough and therefore didn't add more detailed description. Do you think we should extend the description?